### ISSUE TITLE
Evaluate command accuracy very low

### ISSUE THREAD

**Issue by haoyang-udacity** [udacity/robotics-nanodegree-issues#239]
Confusion matrix for both validation and train set show near 100% accuracy.   Why does `evaluate` show only 75%?
Anybody notice that running the supplied data under DIGITS, the validation accuracy can be 100%, but the evaluate command shows the training set accuracy around 75%.  That seems very odd to me, since the validation set should be more or less representative of the data.
